moped-editor/src/views/projects/projectView/ProjectFunding/ProjectFundingTable.js

```diff
@@ -199,7 +209,20 @@ const useColumns = ({
               ${value?.unit_long_name}`
             : "",
         renderEditCell: (props) => (
-          <FundingDeptUnitAutocomplete props={props} value={props.value} />
+          <LookupAutocompleteComponent
+            {...props}
+            name={"dept_unit"}
+            options={deptUnitData}
+            autocompleteProps={{
+              getOptionLabel: (option) =>
+                !!option.dept
+                  ? `${option.dept} | ${option.unit} | ${option.unit_long_name} `
+                  : "",
+              isOptionEqualToValue: (value, option) =>
+                value.unit_long_name === option.unit_long_name,
+            }}
```

Comment:
ditto here. i know i'm probably wringing my hands over nothing, but out of scope variables don't cost a thing.